packages/dara-components/js/common/select/select.tsx

```diff
@@ -175,7 +176,8 @@ function Select(props: SelectProps): JSX.Element {
         // it fail to update the value sometimes. By keeping a local copy of the value and updating it like this we fix
         // the race condition and respect the main value if it is updated elsewhere.
         useEffect(() => {
-            setSelectedItems(getMultiselectItems(value, itemArray));
+            const found = getMultiselectItems(value, itemArray);
+            setSelectedItems(isEmpty(found) ? explicitValues : found);
```

Comment:
I think in the select version we check if explicitValues is undefined as well if so we set it to null, I can't remember why but have you tried to see if that is ok here?

Reply:
I think it was to differentiate undefined/null for controlled component purposes. I'll do the same here to be on the safe side